database/mw/docker-compose-clean.yml

```diff
@@ -3,9 +3,10 @@ version: '3.1'
 services:
 
   mediawiki:
-    image: ghcr.io/wbstack/mediawiki:main
+    image: ghcr.io/wbstack/mediawiki:1.38-7.4-20230316-0
     volumes:
       - ./maintWikWiki.json:/var/www/html/w/maintWikWiki.json
+      - ./:/usr/src/
```

Comment:
I wonder if this mount accidentally overwrites anything the image expects to be there?

Also, `man hier` advises against putting your own stuff in `/usr/src`:

```
       /usr/src
              Source files for different parts of the system, included with some packages for reference purposes.  Don't work here with your own projects, as files below /usr  should  be  read-only
              except when installing software (optional).
```

Would it be a thing to just mount the php script in `maintenance` like this:

```
- ./cleanSql.php:/var/www/html/w/maintenance/cleanSql.php
```

Reply:
Good idea, thanks!